modules/katello_content_view_version.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2018, Sean O'Keeffe <seanokeeffe797@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: katello_content_view_version
+short_description: Create, remove or interact with a Katello Content View Version
+description:
+    - Publish, Promote or Remove a Katello Content View Version
+author: "Sean O'Keeffe (@sean797)"
```

Comment:
I don't think you need quotes for plain strings in yaml. There are several other occurrences, such as L28-29, all the examples (e.g. L96-100 and others). I'd prefer we only use quotes when necessary (such as when the jinja variable is the first thing in a string)